src/components/pages/Offer/OfferCreation/__specs__/OfferCreation.spec.jsx

```diff
@@ -239,6 +308,15 @@ describe('src | OfferCreation', () => {
     })
 
     describe('when creating a new offer', () => {
+      it('should display back link according to redux store data', () => {
+        props.offersSearchFilters = {
+          name: 'searchValue',
+        }
+        const wrapper = mountOfferCreation(props, store)
+        const backButton = wrapper.find('a.back-button')
+        expect(backButton.prop('href')).toMatch('/offres?nom=searchValue')
```

Comment:
```suggestion
        // given
        props.offersSearchFilters = {
          name: 'searchValue',
        }

        // when
        const wrapper = mountOfferCreation(props, store)

        // then
        const backButton = wrapper.find('a[href="/offres?nom=searchValue"]')
        expect(backButton).toHaveLength(1)
```
On a pris pour habitude de ne pas cibler nos éléments avec des noms de class css dans nos tests, parce que c'est un détail d'implémentation et que si on renomme la class, le test ne passe plus alors que la fonctionnalité n'est à priori pas cassée.